src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -143,6 +143,100 @@
     // Finally, set the flag to open the context menu.
     $showContextMenu = true; // eslint-disable-line no-unused-vars
   }
+
+  const getYTicks = (ranges) => {
+    if (
+      data[0].metric_type === 'histogram-linear' ||
+      data[0].metric_type === 'histogram-exponential'
+    ) {
+      // when the range is too small we need to custom set it
+      if (ranges[ranges.length - 1] < 5) return [0, 1, 2, 3, 4, 5];
```

Comment:
lodash has a `_.range(6)` that can return this, but not sure if that's better or not. If it were longer I'd say use lodash, but for just 6 numbers this doesn't seem bad.